packages/include_test_package/lib/analysis_options.yaml

```diff
@@ -0,0 +1,3 @@
+custom_lint:
```

Comment:
Don't create a new project just for one test. Tests should be able to dynamically create projects on the go if necessary.